gossipd/gossip.c

```diff
@@ -1552,7 +1614,11 @@ static struct io_plan *recv_req(struct io_conn *conn, struct daemon_conn *master
 	case WIRE_GOSSIP_GET_UPDATE:
 	case WIRE_GOSSIP_GET_UPDATE_REPLY:
 	case WIRE_GOSSIP_SEND_GOSSIP:
-	break;
+		break;
+
+	/* Sent from channeld through its gossip_fd */
+	case WIRE_GOSSIP_LOCAL_ADD_CHANNEL:
+		break;
 	}
```

Comment:
This is deeply confusing.

Just remove gossip_local_add_channel and the weird call to it: the "break;" here does exactly what we want if the master sends us that message for some reason.

Reply:
Oh wow, seems I botched up a rebase somewhere, sorry about that